t/Net/DRI/Protocol/Whois/Domain/NGTLD.t

```diff
@@ -281,10 +281,10 @@ This information has been prepared and published in order to represent administr
 
 We may discontinue or amend any part or the whole of these Terms of Service from time to time at our absolute discretion.';
 
-$dri->add_registry('NGTLD',{provider=>'ari'});
-$dri->target('ari')->add_current_profile('p1','whois',{f_send=>\&mysend,f_recv=>\&myrecv});
+$dri->add_registry('NGTLD',{provider=>'godaddy_legacy'});
```

Comment:
Is it `godaddy_legacy`? Shouldn't it be `godaddy_dnrs`?

Reply:
no, godaddy legacy uses ari extentions, godaddy_dnrs uses neustar